helm/alfresco-content-services/charts/alfresco-sync-service/README.md

```diff
@@ -40,8 +40,7 @@ Alfresco Syncservice
 | postgresql-syncservice.image.tag | string | `"11.7.0"` |  |
 | postgresql-syncservice.name | string | `"postgresql-syncservice"` | If true, install the postgresql chart alongside Alfresco Sync service. Note: Set this to false if you use an external database. |
 | postgresql-syncservice.nameOverride | string | `"postgresql-syncservice"` |  |
-| postgresql-syncservice.persistence.existingClaim | string | `"alfresco-volume-claim"` |  |
```

Comment:
Do we really want to remove this from the doc?

Reply:
As above